src/js/components/ScaleList.jsx

```diff
@@ -1,8 +1,22 @@
 'use strict';
 var React = require('react'),
+    connect = require('react-redux').connect,
     ContentEditable = require('./ContentEditable');
 
-var ScaleList = React.createClass({
+function mapStateToProps(reduxState, ownProps) {
+  return {
+    selected: reduxState.get('selectedMark')
+  };
+}
+
+var ScaleList = connect(
```

Comment:
I'd rather  we do `module.exports = connect(mapStateToProps,ScaleList);`  just because it's more readable.

Kadam I feel bad having opinions in code reviews.